### PR TITLE
Feature/929 organization cla managers

### PR DESCRIPTION
**Pull request by prasannamahajan**
- adding api to return cla-managers for organization
- adding user_service swagger client
- get company signatures api will return signed and approved signatures only
- adding filter signatureType in get company signatures

### CODE REVIEW

File: cla-backend-go/v2/company/service.go [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+		return nil, err
+	}
+	var claManagers []*models.CompanyClaManager
+	lfUsernames := utils.NewStringSet()
+	projectIDs := utils.NewStringSet()
+	// Get CLA managers
+	for _, sig := range sigs {
+		for _, user := range sig.SignatureACL {
+			claManagers = append(claManagers, &models.CompanyClaManager{
+				ApprovedOn: "",
```
Comment (dealako) [inline cla-backend-go/v2/company/service.go:75]: Just make a note/comment here - DB currently doesn't store the approved on value.

File: cla-backend-go/v2/user-service/client.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+// InitClient initializes the user_service client
+func InitClient(APIGwURL string) {
+	APIGwURL = strings.ReplaceAll(APIGwURL, "https://", "")
+	userServiceClient = &Client{
+		cl: client.NewHTTPClientWithConfig(strfmt.Default, &client.TransportConfig{
+			Host:     APIGwURL,
+			BasePath: "user-service/v1",
+			Schemes:  []string{"https"},
+		}),
+		dummyXACL: "ewogICAgImFsbG93ZWQiOnRydWUsCiAgICAiaXNBZG1pbiI6IHRydWUsICAgIAogICAgInJlc291cmNlIjoidmlld19wcm9qZWN0IiwKICAgICJzY29wZXMiOlsgCiAgICBdIAp9",
```
Comment (dealako) [inline cla-backend-go/v2/user-service/client.go:34]: Offf, magic code alert.  I assume this is to set the ACL header?  If needed, let's externalize this as a SSM parameter or something - not hard-coded in the code.

File: cla-backend-go/v2/company/service.go (hunk 1 above)
Comment (prasannamahajan) [reply cla-backend-go/v2/company/service.go:76]: Done

File: cla-backend-go/v2/user-service/client.go (hunk 2 above)
Comment (prasannamahajan) [reply cla-backend-go/v2/user-service/client.go]: Removed it.

**Pull request merged**